### ISSUE TITLE
Using login authentication method doesn't show any subscriptions

### ISSUE THREAD

**Issue by rtheil-growlink** [Azure/azure-iot-explorer#554]
Using new feature added from #446 doesn't seem to result in any subscriptions showing up for us. I've logged into the app, which works fine, but once I'm logged in, it just says "No subscriptions to show". I've had my boss try this also, and he has all the permissions, and he gets the same result.
![image](https://user-images.githubusercontent.com/96268167/191544614-a470b227-f41b-47ba-8a10-d66defb0c16d.png)

**Comment by YingXue** [comment]
We return the results as is using the following api: https://learn.microsoft.com/en-us/rest/api/resources/subscriptions/list?tabs=HTTP
There is not any filter logic that we are applying.
If you can give this API a try (using postman or anything else), it will give us some hint.

**Comment by YingXue** [comment]
If you'd like to get some answers from the service team, Microsoft Q&A forum will be a good place to ask questions.

**Comment by TechWatching** [comment]
Same issue, @YingXue, when I want to try the API, the tenant needs to be selected. So the issue is indeed not to be able to select the tenant. I agree with @rtheil-growlink , the process should be similar to Azure Storage Explorer which works fine.

**Comment by YingXue** [comment]
Will this post be able to provide a workaround?
https://docs.microsoft.com/en-us/answers/questions/947467/cannot-access-34azure-active-directory34-from-the.html
It looks like users should be able to login with their tenant account like: [user1@yourTenantName.onmicrosoft.com](mailto:user1@yourTenantName.onmicrosoft.com)

**Comment by TechWatching** [comment]
No, it does not because it does not cover the case where your user is a guest on the tenant.

**Comment by rtheil-growlink** [comment]
After finding the correct username for the tenant, I was able to get the login to recognize that username but I was unable to get password accepted for some reason, so I can't log in as the tenant-specific account.

**Comment by mortenn** [comment]
Being able to select the tenant to log into would be a great feature, the IoT Hub I am working with is not part of my home tenant, but lives in a customers separate tenant, thus I am forced to use a connection string to use this tool.

**Comment by rtheil-growlink** [comment]
It's been over 2 years. It would be nice to get some visibility on this issue. Worth repeating that this could just implement the authentication process that's used within Azure Storage Explorer, as it properly handles multiple tenants per user.
